### ISSUE TITLE
bug with default ground truth labeling interface.

### ISSUE THREAD

**Issue by bneigher** [aws-samples/amazon-sagemaker-ground-truth-task-uis#8]
Unsure if this is the place to log it - but for me when inserting a link on the Instructions menu, the link when looking at the active labeling interface doesn't do anything when clicking it.  The only way to open the link is by right clicking the blue link text, and then selecting open in new tab / window.

The issue seems to affect header 1, header 2, and bold text with links as well.

**Comment by dschultz0** [comment]
I'm seeing the same issue. The problem is that they've changed the permissions for the iframe that displays tasks in Ground Truth. They no longer have the 'allow-popups' permission set which is what prevents this. Opening additional tabs to get additional information is a common use case so it's a bit frustrating this doesn't work.

**Comment by lamolivia** [comment]
i agree with this, please let me know if you find a strategy to get past this
